packages/babel-core/test/fixtures/transformation/source-maps/input-source-map-external/input.js

```diff
@@ -0,0 +1,5 @@
+var foo = function () {
+  return 4;
+};
+
+//# sourceMappingURL=input.js.map
```

Comment:
The test case is copied and later transformed from `packages/babel-core/test/fixtures/transformation/source-maps/input-source-map`.